src/queue/extras.rs

```diff
@@ -28,6 +28,8 @@ pub struct Extras {
     pub(crate) fixed_level: Option<u8>,
     /// Number of execute times
     pub(crate) exec_times: u32,
+    /// The task group id. Used in priority queue.
+    pub(crate) group_id: u64,
```

Comment:
why not group name?

Reply:
Done. But I think it a bit weird here because the group_id/name is only fits in some specified cases(e.g. in tikv), but may not fit for other cases. I'm not sure if it is better to be defined as `extra_data: Vec<u8>`, so the user can put arbitary data in this field as they need.